front_end/src/components/voter.tsx

```diff
@@ -43,12 +45,20 @@ const Voter: FC<Props> = ({
         {userVote === 1 ? (
           <FontAwesomeIcon
             icon={faChevronUp}
-            className="rounded-full bg-gradient-to-b from-olive-400 to-blue-100 p-1 text-olive-700 group-hover:from-olive-500 group-hover:to-blue-100 dark:from-olive-300-dark dark:to-blue-100-dark dark:text-olive-700-dark dark:group-hover:from-olive-500-dark dark:group-hover:to-blue-100-dark"
+            className={`from-olive-400 to-blue-100 p-1 text-olive-700 group-hover:from-olive-500 group-hover:to-blue-100 dark:from-olive-300-dark dark:to-blue-100-dark dark:text-olive-700-dark dark:group-hover:from-olive-500-dark dark:group-hover:to-blue-100-dark ${
+              commentArea
+                ? "rounded-none bg-gradient-to-r"
+                : "rounded-full bg-gradient-to-b "
+            }`}
```

Comment:
Let's use `classNames` and comma to combine classes instead of using template string:

```
            className={classNames(
              "from-olive-400 to-blue-100 p-1 text-olive-700 group-hover:from-olive-500 group-hover:to-blue-100 dark:from-olive-300-dark dark:to-blue-100-dark dark:text-olive-700-dark dark:group-hover:from-olive-500-dark dark:group-hover:to-blue-100-dark",
              commentArea
                ? "rounded-none bg-gradient-to-r"
                : "rounded-full bg-gradient-to-b"
            )}
```